i18n/tr.json

```diff
@@ -94,11 +99,19 @@
 	"scratch-confirmaccount-email-resent": "Doğrulama e-postası yeniden gönderildi.",
 	"scratch-confirmaccount-new-scratcher": "Yeni Scratch'çiler hesap isteği gönderemez.",
 	"scratch-confirmaccount-joinedat": "Bir hesap istemek için en az $1 gündür Scratch'te kayıtlı olmanız gerekir.",
+	"scratch-confirmaccount-profile-error": "Hesabınızı kontrol ederken bir hata gerçekleşti. Daha sonra tekrar deneyin.",
 
 	"scratch-confirmaccount-requests-awaiting": "<b>$2</b> $1 $3 bekliyor. <b>İlginiz gerekli!</b>",
-	"scratch-confirmaccount-requests-awaiting-linktext": "{{PLURAL:$1|istek|istek}}",
-	"scratch-confirmaccount-requests-awaiting-isare": "{{PLURAL:$1|bekliyor|bekliyor}}",
+	"scratch-confirmaccount-requests-awaiting-linktext": "istek",
```

Comment:
The correct way to do this is `{{PLURAL:$1|istek}}` - even if there's no other form.

Reply:
ok